tests/cases/struct/LV_HolzapfelOgdenModifiedAnisotropy_passive/svFSIplus.xml

```diff
@@ -58,9 +58,8 @@
    <Density> 1.0 </Density>                           <!-- g/cm^3 -->
    <Elasticity_modulus> 1.0e6 </Elasticity_modulus>   <!-- dyne/cm^2 -->
    <Poisson_ratio> 0.483333 </Poisson_ratio>
-   <Viscosity model="Constant" >
-         <Value> 0.0 </Value>
-      </Viscosity>
+   
+   <!-- <Solid_viscosity> 0.9 </Solid_viscosity> -->
```

Comment:
Small thing, could you remove this commented Solid_viscosity line? I just submitted a pull request updating solid viscosity and this syntax will no longer be valid.